versionalchemy/utils.py

```diff
@@ -165,6 +167,8 @@ class VAJSONEncoder(json.JSONEncoder):
     def default(self, obj):
         if isinstance(obj, datetime.datetime):
             return obj.isoformat()
+        if isinstance(obj, decimal.Decimal):
```

Comment:
Out of interest, why was `Decimal` encoding added to `VAJSONEncoder`?

Reply:
We have a test for encoding decimals as JSON; I’m not totally sure how it worked in Python 2.7

Reply:
Yeah, that’s confusing.

Either way, looks solid. The only thing I notice now is the docstring doesn’t quite match what the class does now. Specifically: https://github.com/NerdWalletOSS/versionalchemy/blob/db5f96e77d8dd35170ffd7a08fe0a6148183bdbc/versionalchemy/utils.py#L160-L163

Reply:
For posterity, the reason this worked before is `simplejson.dumps` handles `Decimal` objects out of the box, while `json.dumps` does not.